src/isomorphic/classic/types/ReactPropTypes.js

```diff
@@ -83,6 +83,7 @@ var ReactPropTypes = {
   oneOf: createEnumTypeChecker,
   oneOfType: createUnionTypeChecker,
   shape: createShapeTypeChecker,
+  symbol: createSymbolTypeChecker(),
```

Comment:
Calling the `createSymbolTypeChecker` returns the chainable type checker, right?
As you would do something like: `React.PropTypes.symbol.isRequired` and not `React.PropTypes.symbol().isRequired` (am I wrong?)